src/configuration/services/solr.md

```diff
@@ -196,3 +196,28 @@ solrsearch:
 ```
 
 The Solr 6.x Drupal 8 configuration files are reasonably generic and should work in many other circumstances, but explicitly defining a core, configuration, and endpoint is generally recommended.
+
+## Accessing the Solr admin interface
```

Comment:
Maybe we better write "Accessing the Solr server's admin interface" to avoid confusion with the framewok's UI admin pages.